.golangci.yml

```diff
@@ -9,8 +9,6 @@ linters-settings:
       - github.com/sirupsen/logrus: "logging is allowed only by logutils.Log"
   dupl:
     threshold: 100
-  exhaustive:
-    default-signifies-exhaustive: false
```

Comment:
Removing because this is just the default value. As @bombsimon said, no need to mention defaults in this file.